x/cdp/client/rest/rest.go

```diff
@@ -1,74 +1,74 @@
 package rest
 
-import (
-	"github.com/gorilla/mux"
+// import (
```

Comment:
to avoid ignoring all files, I tried just creating an empty go.mod file under these folders that not migrated. I think that worked out well and also preserves the original diff once these are migrated. no need to change here i think but something to consider for the future.